e2e/teams.spec.ts

```diff
@@ -137,17 +153,19 @@ test.describe('Teams', () => {
         // ...before resolving to one team link
         await expect
           .poll(async () => {
-            return await teamLink.count()
+            const count = await teamLink.count()
+            // if no team links appear after joining the team
+            // ...try reloading the page to see if the link subsequently appears
+            if (count === 0) {
+              await validatedUserPage.reload()
+            }
+            return count
```

Comment:
Intermittently, the team link does not appear after joining the team on the Sage runner, so try reloading the page if the team link is not visible